src/main/java/eu/greev/dcbot/ticketsystem/interactions/Interaction.java

```diff
@@ -3,5 +3,5 @@
 import net.dv8tion.jda.api.events.Event;
 
 public interface Interaction {
-    void execute(Event event);
+    void execute(Event evt);
```

Comment:
Why not keep the simple `event` name?

Reply:
Because when a class implements this method,   the var name is also `evt`, but I have to cast it always anyways and then I'd like use the var `event` with the correct event type, so at all I changed it because I'm too lazy to change it later by myself